test/recipes.spec.js

```diff
@@ -2,48 +2,63 @@ import chai, { expect } from 'chai';
 import chaiHttp from 'chai-http';
 import server from '../server';
 import { generateToken } from '../server/utils/helpers';
+import { recipe, recipeTestUser } from './fixtures';
 
 chai.use(chaiHttp);
 
-const recipe = {
-  title: 'How to prepare your finest recipe',
-  ingredients: ['A spoon of awesomeness', 'A cup of dedication'],
-  steps: {
-    '1': {
-      description: 'Add a spoon of awesomeness to the mixer',
-      images: ['https://i.stack.imgur.com/xHWG8.jpg']
-    },
-    '2': {
-      description: 'Add a spoon of awesomeness to the mixer',
-      images: ['https://i.stack.imgur.com/xHWG8.jpg']
-    }
-  },
-  cookingTime: 1000,
-  preparationTime: 3000
-};
+const runtimeFixture = {};
+
 describe('Recipes', () => {
+  describe('Fetching before create', () => {
+    it('Get all should return empty array and message when no recipes exist', done => {
+      chai
+        .request(server)
+        .get('/api/recipes')
+        .end((err, res) => {
+          const {
+            body,
+            body: { recipes, message }
+          } = res;
+          expect(res).to.have.status(200);
+          expect(body).to.be.an('object');
+          expect(recipes)
+            .to.be.an('array')
+            .that.has.lengthOf(0);
+          expect(message)
+            .to.be.a('string')
+            .that.eqls('no recipes have been created');
```

Comment:
This test is too explicit. The actual value of the string could change at any time, so there is no need to test for it.

Reply:
I disagree.  I think it is sane to strictly verify outputs when possible. 
I don't think the value of this string will ever change if it does, then this will be updated.